module/networks/list.d.ts

```diff
@@ -8,6 +8,10 @@ interface ContractAddresses {
     readonly superTokenFactory: string;
```

Comment:
Looks good! Just one small thing you need to update the `list.cjs` file under `/main`. You can use `./build.sh` to accomplish this.

Reply:
updated. ty ser